ansible/roles/operator-pipeline/templates/openshift/pipelines/community-hosted-pipeline.yml

```diff
@@ -125,10 +125,34 @@ spec:
         - name: remove-matching-namespace-labels
           value: "true"
 
+    # Set a pending GitHub status check first to provide the user with immediate feedback.
+    - name: set-github-status-pending
+      taskRef:
+        name: set-github-status
+      runAfter:
+        - set-github-started-label
+      params:
+        - name: pipeline_image
```

Comment:
I may not understand this very well, but how does this parameter work? I don't see it in the argument parser in the target script. Where is this passed to? Is it somehow inherited parameter that every task requires?